website/content/docs/commands/node/meta/apply.mdx

```diff
@@ -0,0 +1,35 @@
+---
+layout: docs
+page_title: 'Commands: node meta apply'
+description: |
+  The node meta apply command updates node metadata.
+---
+
+# Command: node meta apply
+
+Modify a node's metadata. This command only applies to client agents, and can
+be used to update the scheduling metadata the node registers.
+
+~> Changes are batched and may take up to 10 seconds to propagate to the
+   servers and affect scheduling.
+
+This command uses the [`/v1/client/metadata` HTTP API][api].
+
+## Usage
+
+```plaintext
+nomad node meta apply -unset testing,tempvar ready=1 role=preinit-db
```

Comment:
```suggestion
nomad node meta apply [options] <key=value>
```

Usually we provide examples in a different section, and leave generic values in `Usage`.

Reply:
Oops, sure enough. Thanks!